nms_stack/nms_cli/nms_stack/roles/nms/templates/docker-compose.yml

```diff
@@ -48,7 +48,7 @@ services:
         max-file: "10"
 
   grafana:
-    image: grafana/grafana:latest
+    image: grafana/grafana:8.5.3
```

Comment:
I think this should be `{{grafana_image}}` so it uses the variable.

Reply:
IIUC the variable only exists in k8_stack? They seem to be done inline for nms_stack (same with efk images)

Reply:
PR updated with image paths extracted